scripts/zones/Lower_Jeuno/npcs/Vingijard.lua

```diff
@@ -12,8 +13,144 @@ require("scripts/zones/Lower_Jeuno/TextIDs");
+if trade:hasItem(65535,10000) then player:completeTrade(); 
+   player:delquest(Jueno,Wings_of_Gold);
+   player:delquest(Jueno,Scattered_into_Shadow);
+   player:delquest(Jueno,A_New_Dawn);
+   player:delquest(Jueno,Borghertzs_Wild_Hands);
+elseif player:getMainJob() = 4 and player:getQuestStatus(Jueno,The_Three_Magi) == QUEST_COMPLETED and player:getQuestStatus(Jueno,The_Recollections) == QUEST_COMPLETED and
+ player:getQuestStatus(Jueno,The_Root_of_the_Problem) == QUEST_COMPLETED and player:getQuestStatus(Jueno,Borghertzs_Sorcereress_Hands) == QUEST_COMPLETED then
+if trade:hasItem(65535,10000) then player:completeTrade(); 
+   player:delquest(Jueno,The_Three_Magi);
+   player:delquest(Jueno,The_Recollections);
+   player:delquest(Jueno,The_Root_of_the_Problem);
+   player:delquest(Jueno,Borghertzs_Sorcereress_Hands);
+--elseif player:getMainJob() = BLU and player:getQuestStatus(Aht_Urhgan,Beginnings) == QUEST_COMPLETED and player:getQuestStatus(Aht_Urhgan,Omens) == QUEST_COMPLETED and
+-- player:getQuestStatus(Aht_Urhgan,Transformations) == QUEST_COMPLETED then
+-- if trade:hasItem(65535,10000) then player:completeTrade(); 
+ --  player:delquest(Aht_Urhgan,Beginnings);
+ --  player:delquest(Aht_Urhgan,Omens);
+ --  player:delquest(Aht_Urhgan,Transformations);
+--elseif player:getMainJob() = COR and player:getQuestStatus(Aht_Urhgan,Equipped_For_All_Occasions) == QUEST_COMPLETED and player:getQuestStatus(Aht_Urhgan,Navigating_the_Unfriendly_Seas) == QUEST_COMPLETED and
+-- player:getQuestStatus(Aht_Urhgan,Against_All_Odds) == QUEST_COMPLETED then
+--if trade:hasItem(65535,10000) then player:completeTrade(); 
+--   player:delquest(Aht_Urhgan,Equipped_For_All_Occasions);
+--   player:delquest(Aht_Urhgan,Navigating_the_Unfriendly_Seas);
+--   player:delquest(Aht_Urhgan,Against_All_Odds);
+--elseif player:getMainJob() = JOBS.DNC and player:getQuestStatus(Jueno,The_Unfinished_Waltz) == QUEST_COMPLETED and player:getQuestStatus(Jueno,The_Road_to_Divadom) == QUEST_COMPLETED and
+-- player:getQuestStatus(Jueno,Comeback_Queen) == QUEST_COMPLETED then
+--if trade:hasItem(65535,10000) then player:completeTrade(); 
+--   player:delquest(Aht_Urhgan,The_Unfinished_Waltz);
+--   player:delquest(Aht_Urhgan,The_Road_to_Divadom);
+--   player:delquest(Aht_Urhgan,Comeback_Queen);
+elseif player:getMainJob() = 8 and player:getQuestStatus(Bastok,Dark_Legacy) == QUEST_COMPLETED and player:getQuestStatus(Bastok,Dark_Puppet) == QUEST_COMPLETED and
+ player:getQuestStatus(Bastok,Blade_of_Evil) == QUEST_COMPLETED and player:getQuestStatus(Bastok,Borghertzs_Shadowy_Hands) == QUEST_COMPLETED then
+if trade:hasItem(65535,10000) then player:completeTrade(); 
+   player:delquest(Bastok,Dark_Legacy);
+   player:delquest(Bastok,The_Recollections);
+   player:delquest(Bastok,The_Root_of_the_Problem);
+   player:delquest(Bastok,Borghertzs_Shadowy_Hands);
```

Comment:
why are you deleting quests? none of the checks here will even work since you're checking if they've been completed